examples/chart/teleport-agent/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+name: teleport-agent
```

Comment:
Is this chart supposed to be Kubernetes service specific (like readme suggests), or can it potentially be extended to support other services as well? Just wondering whether the name is a bit too generic or not.

Reply:
Good point, renamed to `teleport-kube-agent`